apps/login/package.json

```diff
@@ -3,8 +3,19 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "test": "jest",
-    "test:watch": "jest --watch",
+    "test": "concurrently --group --timings --kill-others-on-fail 'npm:test:unit' 'npm:test:integration'",
+    "test:watch": "concurrently --kill-others 'npm:test:unit:watch' 'npm:test:integration:watch'",
+    "test:unit": "jest --config ./__test__/jest.config.ts",
+    "test:unit:watch": "pnpm test:unit --watch",
+    "test:integration": "pnpm mock:build && concurrently --group --names 'mock,test' --success command-test --kill-others 'pnpm:mock' 'ZITADEL_API_URL=http://localhost:22222 start-server-and-test dev http://localhost:3000 \"test:integration:run\"'",
```

Comment:
Because the app build tries to make a connection to the mock server which is not running, yet.
Why does the build try to call the API?

[build:app] ClientError: /zitadel.settings.v2alpha.SettingsService/GetBrandingSettings UNAVAILABLE: No connection established

Reply:
With this https://github.com/zitadel/typescript/pull/28/commits/f396ff3441e8c1a70c2972fae10e18711707fe91, we can run the tests against the prod build, but I think we need to
1. avoid the build step to make API calls
1. concurrently build the mock server and the app bundles before running them (saves a bit time).